File: TG001/Models/Colaborador.cs
using Microsoft.AspNetCore.Identity;
using System.ComponentModel.DataAnnotations;

namespace TG001.Models
{
    public class Colaborador : Usuario
    {

        [Required]
        public string CPF { get; set; }

        [Required]
        public string Nome { get; set; }

        [Required]
        public string CodBanco { get; set; }

        [Required]
        public string Agencia { get; set; }

        [Required]
        public string Conta { get; set; }
    }
}

File: TG001/ClientApp/src/app/Classes/Colaborador.ts
import { Usuario } from "./Usuario";

export class Colaborador extends Usuario {
	constructor(
		public SecurityStamp?: string,
		public PhoneNumberConfirmed?: boolean,
		public PhoneNumber?: string,
		public PasswordHash?: string,
		public NormalizedUserName?: string,
		public NormalizedEmail?: string,
		public LockoutEnd?: Date,
		public LockoutEnabled?: boolean,
		public Id?: string,
		public EmailConfirmed?: boolean,
		public Email?: string,
		public ConcurrencyStamp?: string,
		public AccessFailedCount?: number,
		public TwoFactorEnabled?: boolean,
		public UserName?: string,
		public Matricula?: string,

		public CPF?: string,
		public Nome?: string,
		public CodBanco?: string,
		public Agencia?: string,
		public Conta?: string,

	) {
		super(SecurityStamp, PhoneNumberConfirmed, PhoneNumber, PasswordHash, NormalizedUserName, NormalizedEmail, LockoutEnd, LockoutEnabled, Id, EmailConfirmed, Email, ConcurrencyStamp, AccessFailedCount, TwoFactorEnabled, UserName, Matricula);
	}

}
